app/store/services/activities.js

```diff
@@ -1,20 +1,17 @@
-// constants
-import { API_ENDPOINT, RESOURCES, METHODS } from '../constants/rest';
+import { API_ENDPOINT, RESOURCES, METHODS } from "../constants/rest";
 
-// helper
-import { callJSONApi } from './helpers/api';
+import { callJSONApi } from "./helpers/api";
 
-export const getAthleteActivities = (token) => {
+export const getAthleteActivities = token => {
   const request = {
     endpoint: {
       url: API_ENDPOINT + RESOURCES.ACTIVITIES,
-      httpVerb: METHODS.GET,
+      httpVerb: METHODS.GET
```

Comment:
look like there is something going on with prettier. check , and number of char on a line...